analysis_options.yaml

```diff
@@ -8,9 +8,6 @@ linter:
     avoid_dynamic_calls: false
     use_null_aware_elements: true
 
-Formatter:
```

Comment:
I'd keep this, and actually also include

formatter:
  page_width: 100
  trailing_commas: preserve